sys/include/net/nanocoap.h

```diff
@@ -1956,6 +1959,23 @@ ssize_t coap_build_hdr(coap_hdr_t *hdr, unsigned type, uint8_t *token,
 ssize_t coap_build_reply(coap_pkt_t *pkt, unsigned code,
                          uint8_t *rbuf, unsigned rlen, unsigned payload_len);
 
+/**
+ * @brief   Build empty reply to CoAP request
+ *
+ * This function can be used to create an empty ACK so that a later, separate
+ * response can be sent independently.
+ *
+ * If the request was non-confirmable, this will generate nothing.
```

Comment:
I don't understand, do you mean the 

>  If the request was non-confirmable, this will generate nothing.

part? But this is still true, if no ACK will be generated, the function would just return 0 and write nothing. Or should I add this to `@returns`?

Reply:
my bad